src/http/interceptor/body.js

```diff
@@ -9,7 +9,7 @@ export default function (request, next) {
 
     if (isFormData(request.body)) {
 
-        request.headers.delete('Content-Type');
+        request.headers.delete('Content-Type') || response.headers.get('content-type');
```

Comment:
Hi, It seems to you change the wrong line.   The line number: 40 should be updated. Could you fix it again.